test/api.spec.js

```diff
@@ -11,15 +11,12 @@ describe('API', function() {
   describe('exposed API', function() {
     // plugins might need to access some internal methods from esformatter
     // so we check if these methods are really available
-    var indent = require('../lib/indent');
     var limit = require('../lib/limit');
     var options = require('../lib/options');
 
     it('shoud expose useful methods to plugin authors', function() {
```

Comment:
This looks like it should go away completely eventually? Since you don't want to have plugins have esformatter as a (non-dev) dependency, right?